opendrift/export/io_netcdf.py

```diff
@@ -261,6 +261,8 @@ def import_file_xarray(self, filename, chunks, elements=None):
         self.lonmax = np.float32(self.ds.lon.maxval)
         self.latmax = np.float32(self.ds.lat.maxval)
 
+    self.mode = Mode.Result
```

Comment:
Why do you need this? self.mode should never be modified directly, only through the `@required_mode` decorator.

Reply:
But in this case the simulation is imported from netCDF with Xarray, and data is fed into a new simulation object that is created. Thus it should be necessary in this case to explicitly set mode to Result?

Reply:
Yes, good point.